ops/app/packages.nix

```
@@ -0,0 +1,98 @@
+let
+  inherit (inputs) nixpkgs std self;
+in {
+  server = let
+    chromedriverBin = nixpkgs.fetchurl {
+      url = "https://chromedriver.storage.googleapis.com/102.0.5005.61/chromedriver_linux64.zip";
+      hash = "sha256-SwB82rvinNOBKT3z8odrHamtMKZZWdUY6nJKst7b9Ts=";
+    };
+    src = std.incl self [
+      "build"
+      "packages"
+      "scripts"
+      ".yarn"
+      ".yarnrc.yml"
+      "package.json"
+      "tsconfig.json"
+      "yarn.lock"
+      "yarn-project.nix"
+    ];
+    project = nixpkgs.callPackage (self + /yarn-project.nix) {} {inherit src;};
+
+    replaceLine = regex: replacement: s: let
+      m = builtins.match "(.*\n)${regex}(\n.*)" s;
+    in
+      builtins.concatStringsSep "" [
+        (builtins.elemAt m 0)
+        replacement
+        (builtins.elemAt m 1)
+      ];
+    production-deps = project.overrideAttrs (oldAttrs: {
+      name = "cardano-sdk-production-deps";
+      configurePhase =
+        builtins.replaceStrings
+        ["yarn install --immutable --immutable-cache"]
+        ["yarn workspaces focus --all --production"]
+        oldAttrs.configurePhase;
+      # A bunch of deps build binaries using node-gyp that requires Python
+      PYTHON = "${nixpkgs.python3}/bin/python3";
```

```suggestion
      PYTHON = lib.getExe nixpkgs.python3;
```
Same with others. `rg '/bin/' $PRJ_ROOT`

---

I'd stack that onto https://github.com/input-output-hk/cardano-js-sdk/pull/812#discussion_r1262641927, too.